app/models/piece.rb

```diff
@@ -93,4 +93,20 @@ def remains_on_board?(to_x:, to_y:)
       false
     end
   end
+
+  def successful_capture(to_x:, to_y:)
```

Comment:
**And** it's more concise to omit "successful" . . . so that's fine, I'll shorten the method name.